recipe/meta.yaml

```
@@ -52,16 +56,16 @@ test:
 
 about:
   home: https://github.com/r-barnes/dggridR/
-  license: MIT
+  license: AGPL-3.0-or-later
   summary: Spatial analyses involving binning require that every bin have the same area, but
     this is impossible using a rectangular grid laid over the Earth or over any projection
     of the Earth. Discrete global grids use hexagons, triangles, and diamonds to overcome
     this issue, overlaying the Earth with equally-sized bins. This package provides
     utilities for working with discrete global grids, along with utilities to aid in
     plotting such data.
-  license_family: MIT
+  license_family: AGPL
   license_file:
-    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE.md
```

The AGPL-3.0 license text is distributed with `r-base`, so can use:

```suggestion
    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/AGPL'
```

---

@mfansler it seems like it does not find the license file with this change. What should I do ?

---

Sorry about that, it should be `AGPL-3`.